client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -58,8 +59,8 @@ const ThemesRelatedCard = React.createClass( {
 					{ themes.map( theme => (
 						<li key={ theme.id }>
 							<Card className="themes-related-card__card">
-								<a href={ getDetailsUrl( theme ) }>
-									<img src={ theme.screenshot + '?w=' + '660' }/>
+								<a href={ this.props.getDetailsUrl( theme ) }>
+									<img src={ theme.screenshot + '?w=' + '660' } />
```

Comment:
Maybe there was one historically (that I couldn't remember), but as it stands now, I'll just turn that into '?w=600'. Easy enough to split again should we ever feel the need.